package.json

```diff
@@ -11,6 +11,10 @@
   "bugs": {
     "url": "https://github.com/redbadger/react.london/issues"
   },
+  "scripts": {
+    "start": "make start-production",
+    "postinstall": "make build-production"
+  },
```

Comment:
I don't think it's worth adding a scripts section to package.json if we have the make file - could be confusing.